src/components/MultiHopTrade/components/SpotTradeSuccess/components/YouSavedOrExtra.tsx

```diff
@@ -33,19 +33,21 @@ const cardSx = {
   animation: `${animatedGradient} 15s ease infinite`,
 }
 
-type YouSavedProps = {
-  totalUpsideCryptoPrecision: string
-  totalUpsidePercentage: string
+type YouSavedOrExtraProps = {
+  totalCryptoPrecision: string
+  totalPercentage: string
   sellAsset: Asset
   buyAsset: Asset
+  isExtra?: boolean
```

Comment:
See https://github.com/shapeshift/web/pull/9520/files#r2087537896 - let's not change the signature of this just yet since this will change in the bps PR, but for posteriority:

This component and its parent are already quite a mess mixing two concerns - fees *lack of downside* and delta *upside* - under one umbrella (mb for this monstrocity), so we probably don't want to add more discrimination there 🙏🏽

Reply:
Resolving, see other comment on resolution